integration-tests/src/test/java/oracle/weblogic/kubernetes/ItValidateWebhookReplicas.java

```diff
@@ -0,0 +1,288 @@
+   *           JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+
+    // get a namespace for operator
+    assertNotNull(namespaces.get(0), String.format("Namespace namespaces.get(0) is null"));
+    opNamespace = namespaces.get(0);
+
+    // get a namespace for domain1
+    assertNotNull(namespaces.get(1), String.format("Namespace namespaces.get(1) is null"));
+    domainNamespace = namespaces.get(1);
+
+    // install the operator
+    logger.info("Install an operator in namespace {0}, managing namespace {1}",
+        opNamespace, domainNamespace);
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create a domain resource
+    logger.info("Create model-in-image domain {0} in namespace {1}, and wait until it comes up",
+        domainUid, domainNamespace);
+    createMiiDomainAndVerify(
+        domainNamespace,
+        domainUid,
+        MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG,
+        adminServerPodName,
+        managedServerPrefix,
+        replicaCount);
+  }
+
+  @BeforeEach
+  public void beforeEach() {
+    // check admin server is up and running
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+    // check managed servers are up and running
+    for (int i = 1; i <= replicaCount; i++) {
+      checkPodReadyAndServiceExists(managedServerPrefix + i, domainUid, domainNamespace);
+    }
+  }
+
+  /**
+   * Verify that when a domain and its clusters are running, a kubectl command that attempts to increase
+   * the cluster level replicas to a value more than the max WebLogic cluster size will be rejected
+   * with a clear error message.
+   */
+  @Test
+  @DisplayName("Verify increasing the replicas of a cluster too high will be rejected")
+  void testClusterReplicasTooHigh() {
+    // patch the cluster with replicas value more than max cluster size
+    String patchStr
+        = "["
+        + "{\"op\": \"replace\", \"path\": \"/spec/replicas\", \"value\": 10}"
+        + "]";
+    V1Patch patch = new V1Patch(patchStr);
+    logger.info("Patching cluster resource using patch string {0} ", patchStr);
+    String responseMsg =
+        patchClusterCustomResourceReturnResponse(clusterName, domainNamespace, patch, V1Patch.PATCH_FORMAT_JSON_PATCH);
+    String expectedErrorMsg =
+        "admission webhook \"weblogic.validating.webhook\" denied the request: Change request to cluster resource '"
+            + clusterName
+            + "' cannot be honored because the replica count would exceed the cluster size '5' when patching "
+            + clusterName
+            + " in namespace "
+            + domainNamespace;
+    assertTrue(responseMsg.contains(expectedErrorMsg),
+        "Patching cluster replicas did not return the expected error msg: " + expectedErrorMsg);
+  }
+
+  /**
```

Comment:
Can u specify the exact  max cluster size count set in model file and what is the replica value you are giving in patch command.